massa-pos-exports/src/pos_final_state.rs

```diff
@@ -64,6 +64,14 @@ impl PoSFinalState {
         })
     }
 
+    /// Reset the state of the PoS final state
+    ///
+    /// USED ONLY FOR BOOTSTRAP
```

Comment:
see my comments above on recreating everything instead of making sure we don't forget to reset everything everywhere manually (guaranteed future problems)